src/lockTracer.cpp

```diff
@@ -36,6 +36,16 @@ UnsafeParkFunc LockTracer::_orig_unsafe_park = NULL;
 
 
 Error LockTracer::start(Arguments& args) {
+    if (!VM::loaded()) {
+        return Error("lock tracer only works with the JVM loaded");
+    }
+
+    if (!TSC::frequencyAvailable()) {
+        // shouldn't really happen - frequency should always be available with the JVM loaded.
```

Comment:
This shouldn't happen indeed, and thus the code is redundant.
It may be replaced with `assert`. However, given that `frequencyAvailable()` is not used anywhere else, I'd get rid of this function entirely.